lib/mapper.js

```diff
@@ -22,13 +52,52 @@ exports.track = function(track){
     objCase.del(props, 'revenue');
   }
 
-  return {
-    email: track.email(),
-    userId: track.userId(),
-    eventName: track.event(),
-    createdAt: unixTime(track.timestamp()),
-    dataFields: props
-  };
+  var eventName = track.event();
+  var cart = props["products"] || props["cart"];
```

Comment:
Are event names case sensitive (/ is there an easy way to do case insensitive matches if they're not)? For example "Completed Order" vs "completed order" vs "completedOrder" etc